content/docs/guides/branching-pitr.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Branching — Point-in-time restore (PITR)
+subtitle: Recover your data to previous state using Neon's branching feature
+enableTableOfContents: true
+---
+
+A Neon project has a default 7-day data retention window, which enables creating a branch that reflects the state of your data at a past time. You can use this capability to recover lost data, which is a form of Point-in-time restore (PITR).
+
+This guide provides an example of how to recover your data to point in time before a data loss occurred using Neon's branching feature.
+
+## Create a point-in-time branch
+
+Suppose that you have a table named `orders` that was accidentally deleted by a faulty query. If you know the time you ran the faulty query or when the data loss was first discovered, you can create a point-in-time branch with the data as it existed before the data loss occurred.
+
+To create a point-in-time branch:
+
+1. Navigate to the **Branches** page in the Neon Console.
+1. Click **Create branch** to open the branch creation dialog.
+1. Enter a name for the branch. You can call it `data_recovery`, for example.
+    ![Data recovery create branch dialog](/docs/guides/data_recovery_create_branch.png)
+1. For the **Parent branch**, select the branch where the data loss occurred.
+1. Select the **Time** option to create a branch with data as it existed at a specific date and time. For example, if the data loss occurred on July 11, 2023 at 10:01am, set the time to July 11, 2023, at 10:00am, just before the faulty query was run.
+1. Leave the **Create compute endpoint** option selected. A compute endpoint is required to connect to the new branch.
+1. Click **Create Branch** to create your point-in-time branch.
```

Comment:
or you can `neonctl branches create --name data_recovery --parent 2023-06-11T10:00:00Z` :)